docs/rapidstart.md

```diff
@@ -0,0 +1,717 @@
+    --conf "spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension" \
+    --conf "spark.sql.catalog.spark_catalog=io.unitycatalog.spark.UCSingleCatalog" \
+    --conf "spark.sql.catalog.unity=io.unitycatalog.spark.UCSingleCatalog" \
+    --conf "spark.sql.catalog.unity.uri=http://localhost:8080" \
+    --conf "spark.sql.catalog.unity.token=" \
+    --conf "spark.sql.defaultCatalog=unity"
+```
+
+Initially, this may take a few minutes to run to download the necessary dependencies.  Afterwards, you can run some quick commands to see your UC assets within Spark SQL shell.
+
+### \[Optional\] Running Spark SQL for Cloud Object Stores
+
+If you would like to run this against cloud object storage, the following versions of the `bin/spark-sql` shell command.
+
+#### AWS S3
+
+```
+bin/spark-sql --name "s3-uc-test" \
+    --master "local[*]" \
+    --packages "org.apache.hadoop:hadoop-aws:3.3.4,io.delta:delta-spark_2.12:3.2.1,io.unitycatalog:unitycatalog-spark_2.12:0.2.0" \
+    --conf "spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension" \
+    --conf "spark.sql.catalog.spark_catalog=io.unitycatalog.spark.UCSingleCatalog" \
+    --conf "spark.hadoop.fs.s3.impl=org.apache.hadoop.fs.s3a.S3AFileSystem" \
+    --conf "spark.sql.catalog.unity=io.unitycatalog.spark.UCSingleCatalog" \
+    --conf "spark.sql.catalog.unity.uri=http://localhost:8080" \
+    --conf "spark.sql.catalog.unity.token=" \
+    --conf "spark.sql.defaultCatalog=unity"
+```
+
+#### Azure
+
+```
+bin/spark-sql --name "azure-uc-test" \
+    --master "local[*]" \
+    --packages "org.apache.hadoop:hadoop-azure:3.3.6,io.delta:delta-spark_2.12:3.2.1,io.unitycatalog:unitycatalog-spark_2.12:0.2.0" \
+    --conf "spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension" \
+    --conf "spark.sql.catalog.spark_catalog=io.unitycatalog.spark.UCSingleCatalog" \
+    --conf "spark.sql.catalog.unity=io.unitycatalog.spark.UCSingleCatalog" \
+    --conf "spark.sql.catalog.unity.uri=http://localhost:8080" \
+    --conf "spark.sql.catalog.unity.token=" \
+    --conf "spark.sql.defaultCatalog=unity"
+```
+
+#### GCS
+
+```
+bin/spark-sql --name "gcs-uc-test" \
+    --master "local[*]" \
+    --jars "https://repo1.maven.org/maven2/com/google/cloud/bigdataoss/gcs-connector/3.0.2/gcs-connector-3.0.2-shaded.jar" \
+    --packages "io.delta:delta-spark_2.12:3.2.1,io.unitycatalog:unitycatalog-spark_2.12:0.2.0" \
+    --conf "spark.sql.extensions=io.delta.sql.DeltaSparkSessionExtension" \
+    --conf "spark.sql.catalog.spark_catalog=io.unitycatalog.spark.UCSingleCatalog" \
+    --conf "spark.hadoop.fs.gs.impl=com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystem" \
+    --conf "spark.hadoop.fs.AbstractFileSystem.gs.impl=com.google.cloud.hadoop.fs.gcs.GoogleHadoopFS" \
+    --conf "spark.hadoop.fs.gs.auth.type=ACCESS_TOKEN_PROVIDER" \
+    --conf "spark.hadoop.fs.gs.auth.access.token.provider=io.unitycatalog.spark.GcsVendedTokenProvider" \
```

Comment:
```suggestion
```

We can remove these properties now, as they were added to the connector in [#473](https://github.com/unitycatalog/unitycatalog/pull/473/files#diff-b496bfc650c9a1c585fb5709e010388ca74382670b419be5ebf4faa1bc499dd4R184)